### PR TITLE
Update k8sDnsNodeCache~App~ value

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: helm/default-apps-vsphere/values.yaml [hunk 1]
```diff
@@ -124,7 +124,7 @@ apps:
     # used by renovate
     # repo: giantswarm/etcd-kubernetes-resources-count-exporter
     version: 1.9.0
-  k8sDnsNodeCacheApp:
+  k8sDnsNodeCache:
```
Comment (mcharriere) [inline helm/default-apps-vsphere/values.yaml:127]: schema needs update too

Comment (kopiczko) [reply helm/default-apps-vsphere/values.yaml:127]: Thanks, updated

**Comment by kopiczko** [comment]
Dear @tinkerers-ci, tests passed.

**Comment by kopiczko** [comment]
PTALA

**Review by mcharriere — APPROVED** [review]

**Pull request merged**